pdbtools/pdb_keepcoord.py

```diff
@@ -18,7 +18,7 @@
 """
 Removes all non-coordinate records from the file.
 
-Keeps only MODEL, ENDMDL, END, ATOM, HETATM, and CONECT.
+Keeps only MODEL, ENDMDL, END, ATOM, HETATM, CONECT, and MASTER.
```

Comment:
I would actually remove MASTER lines here as well. All the other ones have coordinate information (or numbering, for CONECT lines) so that's why they are kept.

Reply:
Done